src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('your error message');
+    }
+
+    if (a + b <= c || a + c <= b || c + b <= a) {
+      throw new Error('your error message');
```

Comment:
[CODE STYLE] - The error message should describe what went wrong. Consider something like: 'Triangle inequality theorem violated: the sum of the lengths of any two sides must be greater than the length of the remaining side.'